flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -582,14 +578,9 @@ public enum SchedulerType {
                     .withDescription(
                             Description.builder()
                                     .text(
-                                            "The average size of data volume to expect each task instance to process if %s has been set to %s. "
-                                                    + "Note that since the parallelism of the vertices is adjusted to a power of 2, "
-                                                    + "the actual average size will be 0.75~1.5 times this value. "
-                                                    + "It is also important to note that when data skew occurs or the decided parallelism reaches the %s (due to too much data), "
```

Comment:
I think this note is still needed. 
Maybe with a bit improvement: "It is also important to note that" -> "Note that"